### DISCUSSION TITLE
Airflow on mobile

### DISCUSSION THREAD

**Discussion by thegunner157** [apache/airflow#43471] [Ideas]
Hi, 

Many users are asking questions about viewing Airflow on mobile phones. Is it possible? If so how can we set it up?

**Comment by potiuk** [comment]
Many apps look just fine on mobile phones. Especially the big ones.

Generally Airflow does not have mobile-specific UI. But if you would like to contribute to make our new react UI from Airflow 3 (See https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-38+Modern+Web+Application) to be mobile-enabled, it's a good idea to contribute your work (or maybe pay for) development of mobile-friendly version of it. Even just testing and providing fixes so that it is better on a mobile is a good idea. Also maybe developing a mobile app might be a good idea (though there is almost no mobile app development among the maintainers - so it had to be a 3rd-party).

Aiirflow is created by > 3000 contributors - most of them volunteers - and you can become one of them (also there are companies that pay other developers to contribute).

Since you have many users, that implies you make a good business - and contributing back is the fastest way to get what you want, paying someone to contribute is second-best. Other than that - simply somone will have to add mobile friendly version and it might or might not happen, depending if there is somene who will decide to contribute it.

**Comment by razeghi71** [comment]
Hi @thegunner157,

Regarding mobile access for Airflow, I wanted to mention that I've been working on DagDroid (https://dagdroid.no/), a native Android app designed specifically for Airflow monitoring and management.

For users who need mobile access now, the app provides:
- Native Android interface 
- DAG listing and management
- DAG run monitoring
- Graph View that clearly displays task dependencies and statuses
- Log access
- Task state manipulation
- Support for Google Cloud Composer, MWAA via Cognito Auth, Astronomer and Basic Authentication

I'm currently looking for beta users to test the app and provide feedback. This might be helpful for teams needing mobile access while more integrated mobile-friendly approaches are considered for the official UI.
